crates/cheatcodes/src/inspector.rs

```diff
@@ -871,6 +871,8 @@ impl Cheatcodes {
         let mut known_codes_storage: rHashMap<U256, EvmStorageSlot> = Default::default();
         let mut deployed_codes: HashMap<Address, AccountInfo> = Default::default();
 
+        let test_contract = data.db.get_test_contract_address();
```

Comment:
How is moving this related to zksolc linking? Feel this could have maybe been part of a different PR, but maybe I am missing something.

Reply:
this is related to moving the startup, as it was causing test contract code to be migrated and some validations to fail because of that with mocking
the startup was moved because it caused issues when migration was done in the context of deploying libs
I agree we could have done separate PRs but it was a domino basically

Reply:
same as above